<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeerConsoleWidgetForm</class>
 <widget class="QWidget" name="SeerConsoleWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Seer Console</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <widget class="QPushButton" name="printButton">
     <property name="toolTip">
      <string>Print the log output.</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/seer/resources/RelaxLightIcons/document-print.svg</normaloff>:/seer/resources/RelaxLightIcons/document-print.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="clearButton">
     <property name="toolTip">
      <string>Clear the log output.</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/seer/resources/RelaxLightIcons/edit-clear.svg</normaloff>:/seer/resources/RelaxLightIcons/edit-clear.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="wrapTextCheckBox">
     <property name="toolTip">
      <string>Wrap long lines to the next line.</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="text">
      <string>Wrap</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QAnsiTextEdit" name="textEdit">
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>[program output text]</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="saveButton">
     <property name="toolTip">
      <string>Save the log output to a file.</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/seer/resources/RelaxLightIcons/document-save-as.svg</normaloff>:/seer/resources/RelaxLightIcons/document-save-as.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="fontButton">
     <property name="toolTip">
      <string>Select the log output font.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/seer/resources/icons-icons/font.png</normaloff>:/seer/resources/icons-icons/font.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QHistoryLineEdit" name="stdinLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>100</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="placeholderText">
      <string>[stdin input text]</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QHistoryLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QHistoryLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QAnsiTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">QAnsiTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
